pages/share/[...id].js

```diff
@@ -15,7 +15,7 @@ export default function SharePage() {
   }
 
   return (
-    <Layout>
+    <Layout header={!frame} footer={!frame}>
```

Comment:
The header includes the language toggle button which is important. Why do you need to hide these anyways?

Reply:
I will remove this from current PR. My use case is that I want to put stats in `iframe` in my user panel